fabric-biomes-v1/src/main/java/net/fabricmc/fabric/impl/biome/NetherBiomesImpl.java

```diff
@@ -0,0 +1,42 @@
+/*
+ * Copyright (c) 2016, 2017, 2018, 2019 FabricMC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package net.fabricmc.fabric.impl.biome;
+
+import java.util.Collections;
+import java.util.HashSet;
+import java.util.Objects;
+import java.util.Set;
+
+import net.minecraft.world.biome.Biome;
+
+/**
+ * Sets for internal use only! Stores data that is used by the Nether dimension mixin.
+ */
+public final class NetherBiomesImpl {
```

Comment:
Why make this a class and not add it to `InternalBiomeData`?

Reply:
That is a great question if i'm being honest